pkg/distributor/distributor.go

```diff
@@ -207,19 +259,23 @@ func (d *Distributor) Push(ctx context.Context, req *connect.Request[pushv1.Push
 		for _, raw := range series.Samples {
 			usagestats.NewCounter(fmt.Sprintf("distributor_profile_type_%s_received", profName)).Inc(1)
 			d.profileReceivedStats.Inc(1)
-			d.bytesReceivedTotalStats.Inc(int64(len(raw.RawProfile)))
-			d.bytesReceivedStats.Record(float64(len(raw.RawProfile)))
+			if haveRawPprof {
+				d.metrics.receivedCompressedBytes.WithLabelValues(profName, tenantID).Observe(float64(len(raw.RawProfile)))
```

Comment:
it makes this metrics a bit useless. 

We should try to improve it, may be this should not be supposed to be recorded here ?

Reply:
I dont get it. Why is it useless?
It is just a little bit less accurate for JFR because there is no such thing as `compressedBytes(profName="memory")` because memory is compressed together with cpu/lock/... so we count this metric for JFR as a single profName="JFR"

Reply:
I meant that the metrics now doesn't account for all ingested profile. so I wonder if we should deprecated at some point or change the way it works.

Reply:
it does account for all ingested profiles, just less accurate for jfr(no split by profile type)